backend/src/Manager/StoreOwner/StoreOwnerProfileManager.php

```diff
@@ -118,4 +122,27 @@ public function getStoreOwnerProfileByStoreOwnerId($storeOwnerId): ?StoreOwnerPr
     {
         return $this->storeOwnerProfileEntityRepository->getStoreOwnerProfileByStoreOwnerId($storeOwnerId);
     }
+
+    public function getCompleteAccountStatusByStoreOwnerId($storeOwnerId): ?array
+    {
+        return $this->storeOwnerProfileEntityRepository->getCompleteAccountStatusByStoreOwnerId($storeOwnerId);
+    }
+
+    public function storeOwnerProfileCompleteAccountStatusUpdate(StoreOwnerCompleteAccountStatusUpdateRequest $request): StoreOwnerProfileEntity|string
+    {
+        if($request->getCompleteAccountStatus() !== StoreProfileConstant::COMPLETE_ACCOUNT_STATUS_PROFILE_CREATED && $request->getCompleteAccountStatus() !== StoreProfileConstant::COMPLETE_ACCOUNT_STATUS_PROFILE_COMPLETED) {
```

Comment:
Very good checkpoint, But try in future to use another way like function un const file check this for you.

Reply:
Do you mean a function for checking the value of the field completeAccountStatus? How about using an array of const values in the Constant class and use just the statement "in"?